src/controlplushub.ts

```diff
@@ -31,8 +31,10 @@ export class ControlPlusHub extends LPF2Hub {
     }
 
     protected _currentPort = 0x3b;
+    protected _currentMaxMA = 4175;
```

Comment:
This is the value reported by Hub port info. I tried to verify it and it is probably better than the original value of 2444.
I used http://www.philohome.com/motors/motorcomp.htm as the reference with different motors with no load but it was inconclusive. http://www.philohome.com/wedo2reverse/consumption.htm does not include Control+ hub unfortunately.

Reply:
I confirmed the value of `4715` is using a multimeter.